src/PrestaShopBundle/Service/DataProvider/Marketplace/ApiClient.php

```diff
@@ -46,7 +46,8 @@ public function __construct(
         $locale,
         $isoCode,
         $toolsAdapter,
-        $domain
+        $domain,
+        $shopVersion
```

Comment:
Interesting side effect of this refactoring, this allow to use a very specific version of addons API for our tests: no more bugs because the API for the next version is not set up :-)